src/routers/Router/index.native.js

```diff
@@ -0,0 +1 @@
+export { default } from './Router'
```

Comment:
could we keep routes as they are now (directly in src) just 
a) wrap them in SplashOrRouter (if it's needed) 
b) or wrap before export default if it's needed
?

e.g.
```javascript
// SplashOrRouter.js
import Router from './Router'
import SingupRouter from './SingupRouter'

const [HotRouter, HotSingupRouter] = [Router, SingupRouter]
  .map(router => Platform.select({
    web: hot(router),
    default: router
  })
)
```
or
```javascript
// Router.js
// ...
const Router = (//...
// ...

export default Platform.select({
  web: hot(Router),
  default: Router,
})
```

Reply:
centralised this logic in a utils file to be re-used by any component that needs hot-reloading for web only. fixed.